src/components/Article/Article.css

```diff
@@ -1 +1,31 @@
-/* Put article styles here */
+.article {
+  margin: 5em auto 0;
+  padding: 1em;
+  font-size: 140%;
+  max-width: 800px;
+  background: white;
+  box-shadow: rgba(0, 0, 0, 0.05) 0 3px 15px;
+}
+
+.article__paragraph {
+  margin: 0;
+  color: #333;
+}
+
+.article__paragraph:not(:first-child) {
+  margin-top: 0.7em;
+}
+
+@media (min-width: 500px) {
+  .article {
+    margin: 3.5em auto 0;
+    padding: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .article {
+    margin: 3.5em auto;
+  }
+}
+
```

Comment:
```suggestion
}
```
The problem is still present.
Leave only one newline in the end
![image](https://user-images.githubusercontent.com/96409563/212745352-e036ca94-09f6-4a81-a46f-afbbd72f9076.png)

Reply:
Hope now it's ok. At least run lint doesn't show up any error.